### DISCUSSION TITLE
Workflow templates for personal (not org) users?

### DISCUSSION THREAD

**Discussion by OJFord** [community/community#12443] [Actions]
I'm not sure if this is a 'docs bug' (currently unclear) or a feature request - but I'd really like to use workflow templates for my personal repositories.

[The docs](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization) say:
> Creating starter workflows for your organization
> [...]
> Starter workflows can be created by users with write access to the organization's .github repository. These can then be used by organization members who have permission to create workflows.

But does 'OJFord' count as an organisation here? i.e. can I create a @OJFord/.github repo with workflow templates to use in other @OJFord/* repos?

Edit:
Oh or actually.. maybe they don't make sense for non-orgs because they don't work like I assumed.. I thought they would either be enforced for matching repos, or that they could be used by reference (like actions, but the whole workflow). If they're just suggestions in the UI then that's not really beneficial for personal use.

**Comment by ethomson** [comment]
> But does 'OJFord' count as an organisation here? i.e. can I create a @OJFord/.github repo with workflow templates to use in other @OJFord/* repos?

You can!

> If they're just suggestions in the UI then that's not really beneficial for personal use.

I can imagine that it might avoid some copy-pasta. If I'm a JavaScript developer, for example, I might have a really opinionated set up that I want to codify in a starter workflow template so that I can easily re-use it in multiple repositories.  🤔 

Food for thought!
